lib/outdated.js

```diff
@@ -231,17 +233,24 @@ class Outdated extends ArboristWorkspaceCmd {
           this.maybeWorkspaceName(edge.from)
           : 'global'
 
-        this.list.push({
-          name: edge.name,
-          path,
-          type,
-          current,
-          location,
-          wanted: wanted.version,
-          latest: latest.version,
-          dependent,
-          homepage: packument.homepage,
-        })
+        const include =
+          this.npm.config.get('in-range') === null ||
+          (this.npm.config.get('in-range') === true && wanted === latest) ||
+          (this.npm.config.get('in-range') === false && wanted !== latest)
```

Comment:
So, iiuc, "in-range" means "only show where wanted is latest".  My expectation was that it would be "only show where the wanted version is within my dep range".  Maybe I'm misreading this?

Reply:
So, like, if the current is `1.0.0`, wanted is `1.0.1`, but latest is `2.0.0`, I'd expect `npm outdated --in-range` to show that.

Reply:
Actually, in that case, I'd _also_ expect it to show up for `npm outdated --no-in-range`, because there's an out-of-range version I _could_ install.

Reply:
I wanted a way to filter between the red and the yellow outputs in pretty print.